universal-login-react/src/ui/Modals/ModalWrapper.tsx

```diff
@@ -29,7 +29,7 @@ export const ModalWrapper = ({children, modalClassName, hideModal, message}: Mod
 
   return (
     <div className={useClassFor('modal-backdrop')}>
-      <div className={modalClassName ? `universal-login ${modalClassName}` : 'universal-login universal-login-default'}>
+      <div className={modalClassName ? `universal-login ${modalClassName}` : 'universal-login'}>
```

Comment:
Do we use modalClassName anywhere?

Reply:
Yes, we are using it in LocalStorageBlockedWarningScreen, DashboardModal.